README.md

```diff
@@ -415,6 +417,9 @@ scale step. E.g. 'first', 'second', 'fourth', 'seventh'.
 
 *showOctaves* - A boolean meaning the same as `showOctaves` in `Note#solfege`
 
+#### Scale#signature()
+ - Returns the sequence of notes representing the key signature of a scale
```

Comment:
If you could add an example of how to use it, and which values the method returns, that'd be great!